crates/matrix-sdk/src/sliding_sync/builder.rs

```diff
@@ -40,6 +40,7 @@ pub struct SlidingSyncBuilder {
     rooms: BTreeMap<OwnedRoomId, SlidingSyncRoom>,
     poll_timeout: Duration,
     network_timeout: Duration,
+    restore_pos_from_database: bool,
```

Comment:
I would maybe rename this to `shared_pos`, because that's the intention behind all this: `pos` is shared by all sliding sync instance with the same ID.